packages/@aws-cdk/aws-rds/README.md

```diff
@@ -376,3 +376,51 @@ new rds.OptionGroup(stack, 'Options', {
   ],
 });
 ```
+
+### Serverless
+
+[Amazon Aurora Serverless]((https://aws.amazon.com/rds/aurora/serverless/)) is an on-demand, auto-scaling configuration for Amazon
+Aurora. The databse will automatically start up, shut down, and scale capacity
+up or down based on your application's needs. It enables you to run your database
+in the cloud without managing any database instances.
+
+The following example initializes an Aurora Serverless PostgreSql cluster.
+Aurora Serverless clusters can specify scaling properties which will be used to
+automatically scale the database cluster seamlessly based on the workload. 
+
+```ts
+import * as ec2 from '@aws-cdk/aws-ec2';
+import * as rds from '@aws-cdk/aws-rds';
+
+const vpc = new ec2.Vpc(this, 'myrdsvpc');
+
+const cluster = new rds.ServerlessDatabaseCluster(this, 'AnotherCluster', {
+  engine: rds.DatabaseClusterEngine.AURORA_POSTGRESQL,
```

Comment:
Can you use the non-deprecated version here? We really shouldn't advertise deprecated usage in our README, except possibly as a quick mention with a warning.

Reply:
did you mean the usage of the `unversioned` engine? I can add an example with a specific version to align with recommended usage

re: deprecated - I thought the intention with them was not to mark them as deprecated / walk that change back because it's valid in RDS and we don't want to be opinionated about exposing the capability to set an unversioned database engine

Reply:
I think Shiv also mentioned that it's very common to use the unversioned engines with Serverless, as the instances are fully managed by RDS, so you're less worried about migrations happening "in-place".

I'm fine with un-deprecating the unversioned engine constants if that's what we want.

Reply:
> it's very common to use the unversioned engines with Serverless, as the instances are fully managed by RDS, so you're less worried about migrations happening "in-place".

If that's the case, then I'm fine with the usage in the README, but we definitely need to un-deprecate the unversioned engines; we shouldn't be highlighting/recommending deprecated properties.

Reply:
> > it's very common to use the unversioned engines with Serverless, as the instances are fully managed by RDS, so you're less worried about migrations happening "in-place".
> 
> If that's the case, then I'm fine with the usage in the README, but we definitely need to un-deprecate the unversioned engines; we shouldn't be highlighting/recommending deprecated properties.

agreed, i'll do that in a separate PR (to drop the `@deprecated` from the unversioned database engines